NFIQ2/NFIQ2Algorithm/src/features/FingerJetFXFeature.cpp

```diff
@@ -127,13 +160,15 @@ FingerJetFXFeature::computeFeatureData(
 		    "Could not allocate space for extracted minutiae records.");
 	}
 
-	if (FRFXLLGetMinutiae(hFeatureSet, BASIC_19794_2_MINUTIA_STRUCT,
-		&minCnt, mdata.get()) != FRFXLL_OK) {
+	FRFXLL_RESULT fxResData = FRFXLLGetMinutiae(
+	    hFeatureSet, BASIC_19794_2_MINUTIA_STRUCT, &minCnt, mdata.get());
+	if (fxResData != FRFXLL_OK) {
```

Comment:
For this and others, investigate if we should be using `if (FRFXLL_SUCCESS())` as opposed to comparing against `FRFXLL_OK`.

```suggestion
	if (FRFXLL_SUCCESS(fxResData)) {
```

Reply:
Marked this for review in #176, we won't worry about it here.